src/javascript.js

```diff
@@ -33,7 +33,8 @@ export default class JavaScriptScanner {
           id: message.ruleId.toUpperCase(),
           message: messages[message.ruleId.toUpperCase()].message,
           description: messages[message.ruleId.toUpperCase()].description,
-          code: message.source,
+          code: message.ruleId.toUpperCase(),
```

Comment:
We already use `id` for this, so should we ditch the `id` property? I'm fine with that.